### ISSUE TITLE
White appearance of duodenal mucosa

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11233]
**Preferred term label:**
White appearance of duodenal mucosa

**Synonyms**
Gelée blanche

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/books/NBK578949/ "Endoscopy typically demonstrates a gelée blanche ("white hoar frosting") appearance of the duodenal mucosa"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Chylomicron retention disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001337
- label: White appearance of duodenal mucosa
- parents: [Abnormal duodenum morphology (HP:0002246)](https://hpo.jax.org/app/browse/term/HP:0002246)
- pmid: PMID:33983694
- def: On a typical diet (e.g., no dietary fat restriction), the intestinal mucosa may have a gelee blanche or white hoar frosting appearance on endoscopy.
- comment: This finding may be observed in APOB-related familial hypobetalipoproteinemia. Biopsy of the intestinal epithelium may demonstrate lipid-laden epithelial cells.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11233
- synonyms: Gelee blanche appearance of duodenal mucosa(NONE)|White hoar frosting appearance of duodenal mucosa(NONE)
